index.html

```diff
@@ -146,6 +146,43 @@ <h3>Multi-Post Stories</h3>
         </ul>
         <button class="btn">See Project</button>
     </section>
+    <footer>
+        <h2 id="conttitle">CONTACT US</h2>
+        <p id="conttext">If you have an application you are interested in 
+            developing, a feature that you need built or a project that needs coding.
+             I’d love to help with it </p>
+        <form action="https://formspree.io/f/xvodlvop" method="POST">
```

Comment:
- [ ] Your design is looking great, unfortunately, it is not as close as the Figma design as it could be. Here are two pictures from your project and the design, please consider making some changes!
- Your project: 
![image](https://user-images.githubusercontent.com/73256642/134185501-523cd98a-5654-4ff8-b545-445e696a91b4.png)
- Figma design:
![image](https://user-images.githubusercontent.com/73256642/134185551-b9278aa2-ad1f-4976-a44c-6f804186ef96.png)